magento/pwa-studio
# [PWA-147] Sign Out should revoke customer token using graphql mutation

## Description

App should revoke Customer token on sign out using GraphQL mutation.

## Related Issue





* [[PWA-147](https://jira.corp.magento.com/browse/PWA-147)] Sign Out should revoke customer token using graphql mutation

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Signin with a valid customer/user
2. Grab the oauth token from local storage
3. If you see customer details in the app, the token is confirmed working, but worth sanity checking and running an auth'd `getCustomer` query in a GraphQL client, being sure to append `Authorization: Bearer <token>` header to request
4. Signout - verify you no longer see customer details, and page should refresh and take you back to home
5. Using a GraphQL client, attempt the auth'd `getCustomer` query again. You should now get `The current customer isn't authorized.`

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
